server/services/dashboard/dashboard.go

```diff
@@ -88,6 +98,15 @@ func (t *dashboard) getDashboardGrid() ([][]*schema.DashboardElementView, error)
 				if err != nil {
 					return nil, err
 				}
+			case schema.DashboardElementTypeTCBuilds:
+				tcBuildInfo, err := t.sideStats.TCBuildInfo()
+				if err != nil {
+					return nil, err
+				}
+
+				elementView.BuildStatuses = tcBuildInfo.BuildStatuses
+				elementView.LastChangesAuthor = types.NewNullString(tcBuildInfo.LastChangesAuthor)
```

Comment:
It's better to separate the view interface from interfaces of clients encapsulated in the view provider.
That's could help the view customers to be independent from the view provider realization and if it changes, that won't be matter for the customers.